.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Seeing the warning below, maybe also worth adding a [shellcheck pre-commit hook](https://github.com/koalaman/shellcheck-precommit)

Reply:
Done in 4aed6b0